themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -136,6 +136,7 @@
                 pattern="[0-9]*"
                 value="{$product.quantity}"
                 name="product-quantity-spin"
+                aria-label="{$product.name} {l s='product quantity field' d='Shop.Theme.Checkout'}"
```

Comment:
Shouldn't this start with an uppercase letter?

Reply:
It's the qty field of the product, and if you want to hear something from the screen reader, you should specify which product is linked to this qty field, I think

Reply:
But It made me think about something, I should have added it as a param of the string